Nodejs/Product/Nodejs/Templates/Files/TypeScriptTsConfig/jsconfig.vstemplate

```diff
@@ -0,0 +1,15 @@
+<VSTemplate Version="3.0.0" Type="Item" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005">
+  <TemplateData>
+    <Name Package="{2ffe45c4-5c73-493c-b187-f2e955ff875e}" ID="1044" />
```

Comment:
The ID 1044 and 1045 used for the name and description respectively have not been added yet to the Visual Studio resources. Instead, VS will display the default name for both fields (jsconfig in this case) until the resource name exists.

Reply:
Why don't you add them? The file is in this repo: https://github.com/Microsoft/nodejstools/blob/master/Nodejs/Product/Nodejs/VSPackage.resx#L171

Reply:
My plan was to unify the text in one place for all the jsconfig templates. But since we have decided that for now only to do NodeJS, I have added the changes this repo. We can modify it later if necessary.